extension.json

```diff
@@ -174,7 +174,10 @@
 		},
 		"MirahezeMagicRequestWikiSkins": {
 			"description": "Array. Skins to allow to be selected as the default skin directly from RequestWiki.",
```

Comment:
Perhaps denote that the keys are the internal skin names and the values are the display names?

Reply:
Done